modules/media/ajax/FileUpload.php

```diff
@@ -82,21 +82,23 @@ function uploadFile()
     $config = NDB_Config::singleton();
     $user   =& User::singleton();
     if (!$user->hasPermission('media_write')) {
-        header("HTTP/1.1 403 Forbidden");
+        showMediaError("Permission Denied", 403);
         exit;
     }
 
     // Validate media path and destination folder
     $mediaPath = $config->getSetting('mediaPath');
 
     if (!isset($mediaPath)) {
-        showMediaError("Error! Media path is not set in Loris Settings!");
-        exit;
+        showMediaError(
+            "Media path not set in Loris settings! "
```

Comment:
I know. I tiny request but it always bugs me when LORIS is not all CAP ;)

Reply:
Fixed.. (I didn't write that sentence, I just changed the whitespace of the line and took out the "Error!")

Reply:
Thanks!